src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+
+.calendar {
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $dayBoxGap;
+  width: getCalendarWidth($dayBoxSize, $calendarPadding, $dayBoxGap);
+  padding: 10px;
+
```

Comment:
I propose to use $calendarPadding here and increase vertical paddings, since the top row rises to a height of 20px on hover.

Reply:
Thx for advice